test/all_parameter_combs.py

```diff
@@ -43,6 +43,7 @@
     (knights_archers_zombies_v8.env, dict(line_death=False)),
     (knights_archers_zombies_v8.env, dict(pad_observation=False)),
     (knights_archers_zombies_v8.env, dict(max_cycles=100)),
+    (knights_archers_zombies_v8.env, dict(vector_state=True)),
```

Comment:
Can you also include the other new parameters here?

Reply:
Included in the latest commit.